doc/website/blog/2024-01-24-why-schema-as-code.mdx

```diff
@@ -0,0 +1,150 @@
+---
+title: "Why Your Team Needs a Database Schema-as-Code Tool"
+authors: rotemtam
+tags: [database-schema-as-code, devops, ci/cd, migrations, terraform, kubernetes, gitops]
+---
+
+### The Evolution of Database Schema Management
+
+In today's world, where software is developed and deployed at a rapid pace, selecting the right tools for your team
+is crucial. The right tools can help you move faster, be more productive, and reduce the risk of errors and bugs.
+Most backend applications are backed by a database, and maintaining the database schema is a critical part of the
+development process.
+
+Our industry has seen many trends in the way database schemas evolve over the years. In the early days, DBAs would
+manually run SQL DDL commands directly against the database through a terminal.  Then came GUI based, point-and-click
```

Comment:
```suggestion
manually run SQL DDL commands directly against the database through a terminal.  Then there were GUI based solutions, point-and-click
```

Reply:
gui based, describes the point and click tools, so i disagree here